client/prisonbreak.lua

```diff
@@ -31,6 +31,10 @@ end
 -- Events
 
 RegisterNetEvent('electronickit:UseElectronickit', function()
+    if QBX.PlayerData.job.type == 'leo' and QBX.PlayerData.job.onduty < Config.minPolice then
```

Comment:
This code doesn't make sense.
First of all, you're comparing a boolean to a number.
Second of all, this code is client side and only checking one person, this should be server side with a callback.